components/user_settings/security/user_access_token_section/user_access_token_section.tsx

```diff
@@ -296,19 +328,19 @@ export default class UserAccessTokenSection extends React.PureComponent {
             );
         }
 
-        const tokenList = [];
+        const tokenList: JSX.Element[] = [];
         Object.values(this.props.userAccessTokens).forEach((token) => {
             if (this.state.newToken && this.state.newToken.id === token.id) {
                 return;
             }
 
-            let activeLink;
+            let activeLink: JSX.Element;
             let activeStatus;
 
             if (token.is_active) {
                 activeLink = (
                     <a
-                        name={token.id + '_deactivate'}
+                        data-name={token.id + '_deactivate'}
```

Comment:
Why was this attribute name changed?

Reply:
> Why was this attribute name changed?

I got error: 
```js
Type '{ children: Element; name: string; href: string; onClick: (e: MouseEvent<HTMLAnchorElement, MouseEvent>) => void; }' is not assignable to type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.
  Property 'name' does not exist on type 'DetailedHTMLProps<AnchorHTMLAttributes<HTMLAnchorElement>, HTMLAnchorElement>'.

(JSX attribute) name: string
```
so I fixed this by adding `data-*`

Reply:
Hmm, looks like it should probably be an `id`. Can we change it to that?

Reply:
I changed, and it works, thank you. There is an issue, I ran type check in my local and it passed, however, CI shows the error at the unrelated files. https://app.circleci.com/pipelines/github/mattermost/mattermost-webapp/28734/workflows/8bc8969c-bec9-44ed-8b12-b918b38427bc/jobs/189515

Reply:
@hiendinhngoc Just had to update your branch, looks like checks are passing now :)